politeiawww/www.go

```diff
@@ -608,6 +608,50 @@ func (p *politeiawww) handleResetPassword(w http.ResponseWriter, r *http.Request
 	util.RespondWithJSON(w, http.StatusOK, rpr)
 }
 
+// handleProposalAccessTime returns the proposal access time for a given user
+func (p *politeiawww) handleProposalAccessTime(w http.ResponseWriter, r *http.Request) {
+	log.Tracef("handleProposalAccessTime")
+	user, err := p.getSessionUser(w, r)
+	if err != nil {
+		RespondWithError(w, r, 0,
+			"handleProposalAccessTime: getSessionUser %v", err)
+		return
+	}
+	reply, err := p.backend.ProcessProposalAccessTime(user.Email)
```

Comment:
I see from the route definition that the proposal token is passed as a parameter however that token value is never accessed or used.